### PR TITLE
feat: QR-based add NGRAVE ZERO Hardware

### PR DESCRIPTION
**Pull request by xardass**
## **Description**
Hello,

This PR changes the UI and text to add the NGRAVE ZERO hardware wallet.

Our upcoming release planned for the beginning of July will add QR code compatibility with MetaMask following EIP-4527.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25080?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/536

## **Manual testing steps**

1. On the hardware wallet, open the 'sync' screen and choose MetaMask
2. Add a QR based wallet on MetaMask (notice that NGRAVE is not 'coming soon' anymore, but listed as a normal option)
3. Scan the QR code from the hardware wallet.

### **Before**
image

### **After**

image

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by AlexJupiter** [comment]
@xardass let's not have this in block capitals, could you change that please?

**Comment by xardass** [comment]
> @xardass let's not have this in block capitals, could you change that please?

Hi Alex,

Thanks for the quick review and response! We get your request about the capitalization, but it’s a key part of our branding and trademark. Our name and products are always in caps:

NGRAVE ZERO
NGRAVE GRAPHENE
NGRAVE LIQUID

You can check out our branding/press kit here: [NGRAVE Press Kit](https://ngrave.io/en/press/).
We’d really appreciate your help in keeping our brand consistent on MetaMask. 

Thanks!

**Comment by xardass** [comment]
@AlexJupiter we changed the capitalization, removed full block capitals.

**Review by Akaryatrh — APPROVED** [review]

**Comment by Akaryatrh** [comment]
LGTM

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 1]
```diff
@@ -788,11 +788,41 @@ export default class SelectHardware extends Component {
...
+              variant={BUTTON_VARIANT.SECONDARY}
+              onClick={() => {
+                this.context.trackEvent({
+                  category: MetaMetricsEventCategory.Navigation,
+                  event: 'Clicked Ngrave Learn more',
+                });
+                openWindow(HardwareAffiliateTutorialLinks.ngrave);
+              }}
+              data-testid="ngrave-brand-learn-more-btn"
+            >
```
Comment (gantunesr) [inline ui/pages/create-account/connect-hardware/select-hardware.js:821]: @AlexJupiter are these events relevant for tracking?

Comment (AlexJupiter) [reply ui/pages/create-account/connect-hardware/select-hardware.js:821]: Yeah these events will be useful. I don't think this is actually following our data schema, but they are they same format as all the other events on this page so I think it's fine for now.

**Review by gantunesr — APPROVED** [review]

**Pull request merged**